firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3700,6 +3700,156 @@ extension String {
+            comment: "Header for the enhanced tracking protection screen when the user has selected to be protected but the connection is not secure.")
+
+        public static let connectionVerifiedByLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.Verifier.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Verified by %@",
+            comment: "String to let users know the site verifier, where the placeholder represents the SSL certificate signer.")
+
+        public static let trackersBlockedLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.Trackers.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "%i Trackers blocked",
+            comment: "String to let users know how many trackers were blocked on the current website.")
+
+        public static let noTrackersLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.Trackers.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "No trackers found",
+            comment: "String to let users know that no trackers were found on the current website.")
+
+        public static let crossSiteTrackersBlockedLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.Trackers.CrossSite.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "%@ Cross-site tracking cookies",
+            comment: "String to let users know how many cross-site tracking cookies were blocked on the current website.")
+
+        public static let socialMediaTrackersBlockedLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.Trackers.SocialMedia.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "%@ Social media tracker",
+            comment: "String to let users know how many social media trackers were blocked on the current website.")
+
+        public static let fingerprinterBlockedLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.Trackers.Fingerprinter.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "%@ Fingerprinter",
+            comment: "String to let users know how many fingerprinters were blocked on the current website.")
+
+        public static let connectionSecureLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.ConnectionSecure.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Secure Connection",
+            comment: "String to let users know that the current website is secure.")
+
+        public static let connectionUnsecureLabel = MZLocalizedString(
+            key: "Menu.EnhancedTrackingProtection.Details.ConnectionUnsecure.v128",
+            tableName: "EnhancedTrackingProtection",
+            value: "Connection is not secure",
```

Comment:
In the Figma, I see `Connection not secure`. Not sure what the latest copy should be.